package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Watches package repo RSS feeds (e.g. PyPi and Maven) and pushes incoming changes to a RabbitMQ event queue",
   "main": "src/server.js",
   "scripts": {
+    "lint": "eslint src util",
```

Comment:
`eslint .` would be better, since we should be enforcing our standards on all files (including tests).